Revit_Core_Engine/Convert/Environment/FromRevit/CableTray.cs

```diff
@@ -94,6 +152,20 @@ public static List<BH.oM.MEP.System.CableTray> CableTrayFromRevit(this Autodesk.
             return bhomCableTrays;
         }
 
+        /***************************************************/
+        /****              Private Methods              ****/
+        /***************************************************/
+        
+        [Description("Re-orders converted BHoM Line list to match the direction from the reference MEPCurve.")]
+        private static List<BH.oM.Geometry.Line> MatchRevitOrder(List<BH.oM.Geometry.Line> linesToMatch, MEPCurve reference)
```

Comment:
Could you please embed this method inside `LocationCurveMEP`? I believe this should be a standard, not cabletray-specific.